static/ankersrv.css

```diff
@@ -1,41 +1,79 @@
 .nav-link,
 .btn-link {
-  color: #88f387;
-  text-decoration: none;
+    color: #88f387;
+    text-decoration: none;
 }
 
 .nav-link.active,
 .btn-primary {
-  color: #000000 !important;
-  background-color: #88f387 !important;
-  border: #41a03f;
+    color: #000000 !important;
+    background-color: #88f387 !important;
+    border: #41a03f;
 }
 
 .nav-link:hover,
 .btn:hover {
-  color: #000000;
-  background-color: #adf3ac;
-  border-color: #41a03f;
+    color: #000000;
+    background-color: #adf3ac;
+    border-color: #41a03f;
 }
 
 a.text-light {
-  text-decoration: none;
+    text-decoration: none;
 }
 
 a.text-light:hover {
-  color: #88f387 !important;
+    color: #88f387 !important;
 }
 
 pre,
 code {
-  color: #88f387;
-  background-color: #292929;
+    color: #88f387;
+    background-color: #292929;
 }
 
 .form-label {
-  font-weight: bolder;
+    font-weight: bolder;
 }
 
 code#loginFilePath {
-  font-weight: lighter;
+    font-weight: lighter;
+}
+
+.print-status {
+    font-size: medium;
+}
+
+.progress {
+    height: 40px;
+}
+
+.icon {
+    background-size: cover;
+    display: inline-block;
+    position: fixed;
+    height: 25px;
+    width: 25px;
+}
+
+.icon-hotend {
+    background-color: #fff;
+    -webkit-mask-image: url(../static/img/hotend-icon.svg);
+    mask-image: url(../static/img/hotend-icon.svg);
+}
+
+.icon-bed {
+    background-color: #fff;
+    -webkit-mask-image: url(../static/img/bed-icon.svg);
+    mask-image: url(../static/img/bed-icon.svg);
```

Comment:
They are black SVGs, I did this here so that I could override the color without having to modify the SVG itself and found no other good way to do this.

Reply:
Ah, that makes sense. Thanks for the explanation :)